scipy/optimize/_minimize.py

```diff
@@ -961,3 +972,42 @@ def standardize_constraints(constraints, x0, meth):
                 constraints.extend(old_constraints[1:])  # appends 1 if present
 
     return constraints
+
+
+def _optimize_result_for_equal_bounds(
+        fun, bounds, method, args=(), constraints=()
+):
+    """
+    Provides a default OptimizeResult for when a bounded minimization method
+    has (lb == ub).all().
+
+    Parameters
+    ----------
+    fun: callable
+    bounds: Bounds
+    method: str
+    constraints: Constraint
+    """
+    success = True
+    message = 'Optimization terminated successfully.'
+
+    # bounds is new-style
+    x0 = bounds.lb
+
+    if constraints:
+        constraints = standardize_constraints(constraints, x0, 'new')
+
+    for c in constraints:
+        maxcv = 0
```

Comment:
```suggestion
    maxcv = 0
    for c in constraints:
```

Reply:
I was thinking of this very thing, make sure it's outside the loop, I don't know how I overlooked that.